oneanddone/tasks/templates/tasks/task_list_filters.html

```diff
@@ -6,8 +6,10 @@
       <fieldset>
         <legend class="visually-hidden">{{ _('Select filters below to filter tasks.') }}</legend>
         <p>
+          <div title ="{{_('Filter on task name, description, or keywords.') }}">
```

Comment:
Missing space between `{{` and `_('Filter on task name, description, or keywords.')`.